hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -206,29 +213,57 @@ def lookup_get():
      Also (applies to both cases):
      * Optional seed (content id) for consistent coinflip
     Output:
-     * JSON object with a key of each bank name with matches.
-       For each bank, there is list of objects containing details
-       about each match.
+     * JSON object keyed by signal type, to a JSON object of bank
+       matches. If Signal Type is given, the outer JSON object is
+       elided.
```

Comment:
This formalizes what had been sort of happening already, but I'm generally skeptical of these kinds of behaviors.

Reply:
no preferences on my side (though we are using the outer-level bank response already)